langgraph/src/channels/base.ts

```diff
@@ -77,15 +77,15 @@ export function emptyChannels(
   return newChannels;
 }
 
-export async function createCheckpoint<ValueType>(
```

Comment:
This function needs to be non-async in order for `_localRead()` to be patched in `RunnableConfig` and eventually called so it doesn't return a `Promise`.

Need to double check if this is ok.

Reply:
yep I think this doesn't have to be async